people/silva-zuur.yml

```diff
@@ -20,3 +20,12 @@ relationships:
       - '@type': Role
         type: roleTypes/group
         agent: groups/enspiral-services
+  - '@type': Relationship
+    type: relationshipTypes/membership
+    roles:
+      - '@type': Role
+        type: roleTypes/member
+        <<: *me
```

Comment:
alias `me` is not defined within Silvia. sorry if the `me` usage is confusing, Mix used it for his data entry but i prefer not using it for the data entry i did.